libcylinder/Cargo.toml

```diff
@@ -39,6 +39,7 @@ experimental = [
   # The following features are experimental:
 ]
 
+hash = ["openssl"]
```

Comment:
Openssl seems like a pretty heavy dependency just for hashing.  I'd be open to converting this to using the [sha2](https://crates.io/crates/sha2) crate. (Can be left to a future PR).

Reply:
I'll make this change in a follow-up PR